modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -163,6 +163,17 @@ For request headers, these adjustments are applied only for routes that have the
 * `threadCount` specifies the number of threads to create per HAProxy process. Creating more threads allows each Ingress Controller pod to handle more connections, at the cost of more system resources being used. HAProxy
 supports up to `64` threads. If this field is empty, the Ingress Controller uses the default value of `4` threads. The default value can change in future releases. Setting this field is not recommended because increasing the number of HAProxy threads allows Ingress Controller pods to use more CPU time under load, and prevent other pods from receiving the CPU resources they need to perform. Reducing the number of threads can cause the Ingress Controller to perform poorly.
 
+* `clientTimeout` specifies how long a connection will be held open while waiting for a client response. If unset, the default timeout is 30s.
+
+* `serverFinTimeout` specifies how long a connection will be held open while waiting for the server response to the client that is closing the connection. If unset, the default timeout is 1s.
+
+* `serverTimeout` specifies how long a connection will be held open while waiting for a server/backend response. If unset, the default timeout is 30s.
```

Comment:
> In most cases, do not use slashes in text because they can be ambiguous.

> In a few cases, use a forward slash (/) in terms where the slash construction is an industry standard.

This might be fine, but I'm not sure that this is a standard term.

Reply:
It should be reasonable to replace "server/backend" with "server" here as well as in the description of `clientFinTimeout`